website/src/pages/v1/serve/features/monitoring-tracing/prometheus.mdx

```diff
@@ -4,78 +4,752 @@ description:
+filter is to include only `statusCode` with `200` value and `method` with `POST` (the default method
+for GraphQL requests, but it can also be `GET` depending on your client setup) value to get
+execution time of successful GraphQL requests only.
+
+To enable this metric, set the `http` option to `true`.
+
+#### Labels
+
+This metric includes some useful labels to help you identify requests and group them together.
+
+| Label           | Enabled by default | Description                                                                                                                                                                                                                                                                                                                                                                             |
+| --------------- | :----------------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `method`        |  Not configurable  | The HTTP method used to request the gateway endpoint.<br/><br/> Since GraphQL usually only uses `POST` requests, this can be used to filter out GraphiQL-related requests. <br/><br/> It can be any HTTP verb, including disallowed ones. Which means this metric can also be used to track malformed or malicious requests.                                                            |
+| `statusCode`    |  Not configurable  | The HTTP status code returned by the gateway.<br/><br/>You probably want to filter out non-`200` responses to have a view of the successful requests.<br/><br/>This can help you identify which requests are failing and why. Since GraphQL errors are returned as `200 OK` responses, this can be useful to track errors that are not related to the GraphQL, like malformed requests. |
+| `operationName` |       `true`       | If available, the name of the GraphQL operation requested, otherwise `Anonymous`.<br/><br/>This can help you identify which operations are slow or failing.<br/><br/>We recommend you always provide an operation name to your queries and mutations to help performance analysis and bug tracking.                                                                                     |
+| `operationType` |       `true`       | The type of the GraphQL operation requested. It can be one of `query`, `mutation`, or `subscription`.<br/><br/>This can help you differentiate read and write performance of the system. It can for example help understand cache impact.                                                                                                                                               |
+
+#### Sample Output
+
+```
```

Comment:
``````suggestion
```text
``````

Reply:
It changes something ? I thought it was the default ^^'